openai/README.md

```diff
@@ -2,19 +2,19 @@
 
 ## Overview
 
+Monitor, troubleshoot, and evaluate your LLM-powered applications, such as chatbots or data extraction tools, using [OpenAI][1]. With LLM Observability, you can investigate the root cause of issues, monitor operational performance, and evaluate the quality, privacy, and safety of your LLM applications.
+
+[LLM Obs tracing view video][16]
+
 Get cost estimation, prompt and completion sampling, error tracking, performance metrics, and more out of [OpenAI][1] account-level, Python, Node.js, and PHP library requests using Datadog metrics, APM, and logs.
 
 ## Setup
+<!-- xxz tab xxx -->
+<!-- xxx tab "API Key" xxx -->
 
-<!-- xxx tabs xxx -->
-<!-- xxx tab "Python" xxx -->
-
-**Note**: This setup method does not collect `openai.api.usage.*` metrics. To collect these metrics, also follow the API key setup instructions.
+**Note**: This setup method only collects `openai.api.usage.*` metrics. To collect all metrics provided by this integration, also follow the APM setup instructions.
 
 ### Installation
-
-#### Web: Get Account-level Usage and Cost Metrics
```

Comment:
why did we remove this? This also shows up on the tile right? 

We want to keep this with a slight change: `#### API Key: Get Account-level Usage and Cost Metrics`

Reply:
You mentioned that the api section should be the only place that contains api key instructions.
The tile uses custom code independent of the setup instructions here. So this only impacts the public docs.

Reply:
Ok that makes senses, thought this would also update the tile.  

Is this the PR for the tile changes? https://github.com/DataDog/web-ui/pull/165662/files If so, I don't see any updates to there related to API Key. Both the docs and the tile should mirror each other.